source/read/retrieve.txt

```diff
@@ -168,6 +169,36 @@ For a full list of available arguments, see the
 `API documentation <{+api-root+}pymongo/collection.html#pymongo.collection.Collection.find>`__
 for the  ``find() method``.
 
+.. _pymongo-retrieve-collation:
+
+Collation
+`````````
+
+When you perform a query, you can specify a **collation** for the driver to follow when
+sorting the results.
+
+.. include:: /includes/collation-description.rst
+
+The following example performs the same find operation as the previous example,
+but with a default collation of ``fr_CA``:
+
+.. code-block:: python
+   :copyable: true
+
+   cursor = sample_restaurants.restaurants.find({"cuisine": "Italian"}, max_time_ms=10000,
+                                              collation=Collation(locale="fr_CA"))
```

Comment:
S: consider moving into the file and including?

Reply:
we'll likely do this for all code samples as part of comprehensive coverage; i'll wait until then so know the final organization of code files